src/Radarr.Api.V3/Indexers/ReleaseResource.cs

```diff
@@ -51,8 +50,6 @@ public class ReleaseResource : RestResource
         public int? Leechers { get; set; }
         public DownloadProtocol Protocol { get; set; }
 
-        // Sent when queuing an unknown release
-        [JsonProperty(DefaultValueHandling = DefaultValueHandling.Ignore)]
```

Comment:
They added further options for this in net5 (https://github.com/dotnet/runtime/issues/779)

Should we hold off on all the serialization changes until after a 3.0, seems kinda big to change json library for API if its only for minimal gains (based on what you mentioned about the other items being the bulk) until 3.1 (net 5)

Reply:
Might be best to leave this for net 5